src/azure-cli/azure/cli/command_modules/ams/_help.py

```diff
@@ -4,744 +4,594 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-from knack.help_files import helps  # pylint: disable=unused-import
 # pylint: disable=line-too-long, too-many-lines
 
+from knack.help_files import helps
```

Comment:
Why do you add so many changes in _help.py?

Reply:
These changes were made by a contractor. It appears that he changed the order of some command/sub commands and changed the spacing to match what he saw on the screen.
If you require it, I can find the version of the file without all these changes and just apply the text changes.

Reply:
@Juliehzl do you want me to undo the changes made to _help.py that were just re-ordering the help/spacing?

Reply:
@Juliehzl , can you please review. Thanks

Reply:
It's fine for me. But please make sure no help missing for your commands.

Reply:
But spacing is not necessary.

Reply:
Fixed spacing.